### DISCUSSION TITLE
Track repeats

### DISCUSSION THREAD

**Discussion by Rogerwalker259** [AzuraCast/AzuraCast#4301] [General]
i'm using Auto Dj
I have started to get random tracks being repeated back to back. The repeated track shows as only playing once in the song playback time line.
This problem happens maybe two or three times a day
Any body else having this problem

**Comment by Marcioghiraldelli** [comment]
I noticed a problem in assembling the playlist, random and sequential. From what I understand, azura mounts the list when scheduled next to run, if you put sequentially it often doesn't mount in the right sequence or if mounts doesn't put all the files. The mode managed by azuracast only works well if you are going to leave without many schedules. So I myself am currently not using the managed mode by azuracast but by liquidsoap which does not repeat songs and follow the sequence of songs correctly, the only problem with this mode is that sometimes it bugs and does not show the name of the playlist that is being executed and yes another one that is totally out of time, but it follows the list correctly.

**Comment by Rogerwalker259** [comment]
Hi I m running in liquidsoap mode and set to random, and still get the
occasional track repeat
